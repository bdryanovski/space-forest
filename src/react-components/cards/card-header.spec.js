import React from 'react'
import renderer from 'react-test-renderer'
import CardHeader from './card-header'

describe('CardHeader', () => {
  it('should match default snapshot', () => {
    const component = renderer.create(<CardHeader>Hello</CardHeader>)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
