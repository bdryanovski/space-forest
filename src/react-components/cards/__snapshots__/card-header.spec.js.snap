// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader should match default snapshot 1`] = `
<div
  className="card-header"
>
  Hello
</div>
`;
